coordinator/src/db/liquidity_options.rs

```diff
@@ -37,16 +35,6 @@ pub(crate) fn get_all(conn: &mut PgConnection) -> QueryResult<Vec<commons::Liqui
     Ok(options)
 }
 
-pub(crate) fn get(
```

Comment:
❓ did you delete this on purpose? don't we need it again when we are introducing liquidity options?

Reply:
Good catch, I think we can still keep this.